qcore/testing.py

```diff
@@ -0,0 +1,47 @@
+import sys
+import os
+from shutil import rmtree
+
+from qcore import shared
+
+
+def test_set_up(realizations):
```

Comment:
Maybe some docstring for a brief overview of what this function does? And where its used?